internal/scaffold/ansible/entrypoint.go

```diff
@@ -45,5 +45,6 @@ if ! whoami &>/dev/null; then
   fi
 fi
 
-exec ${OPERATOR} run ansible --watches-file=/opt/ansible/watches.yaml $@
+cd $HOME
```

Comment:
Is this necessary? We should be able to set the CWD to $HOME in the dockerfile, right?

Reply:
It is required to be sure that the user will be in the HOME when will start the image which by default is `/opt/ansible/`.